IPython/__init__.py

```diff
@@ -47,9 +47,7 @@
 from .core.application import Application
 from .terminal.embed import embed
 
-from .core.error import TryNext
 from .core.interactiveshell import InteractiveShell
-from .testing import test
```

Comment:
I did consider each import given by pyflakes individually, and did remove both on purpose, but if you prefer to keep test, then fine. I'll add it back.